frontend/app/services/lookup-service.server.ts

```diff
@@ -376,6 +376,19 @@ function createLookupService() {
     return federalSocialPrograms;
   }
 
+  function getFederalSocialProgramsById(id: string) {
+    log.debug('Fetching federal social program with id: [%j]', id);
+
+    const federalSocialProgram = getAllFederalSocialPrograms().find((program) => program.id === id);
+
+    if (!federalSocialProgram) {
+      throw new Error('Failed to find federal social program');
```

Comment:
```suggestion
      throw new Error(`Failed to find federal social program; id: ${id}`);
```

Same for `getProvincialTerritorialSocialProgramById`